MVVMIOS/Models/Response.swift

```diff
@@ -7,6 +7,44 @@
 
 import Foundation
 
-struct Response: Codable {
+struct Response: Codable, Equatable {
     let name: String
 }
+
+struct News<T: Codable>: Codable {
+    let data: T
+}
+
+struct Pagination: Codable {
+
+}
+
+struct NewsData: Codable {
+    let id: UUID = UUID()
+    let author: String?
+    let title: String?
+    let description: String?
+    let url: String?
+    let source: String?
+    let category: String?
+    let language: String?
+    let country: String?
+
+    enum CodingKeys: String, CodingKey {
+        case id
+        case author, title, description, url, source, category, language, country
+    }
```

Comment:
You don't have to include coding keys if you aren't changing the names of any of them. I think you can just delete this.
```suggestion
```

Reply:
@tinder-owenthomas I can say that when using defaults in codable, we need to explicitly add coding keys. That was the reason for me to add coding keys as well. I will see if your aforementioned comment allows me to add a default value.

Reply:
Does the default value the way you have it written actually work with Codable though? I think the only way to default items in a Codable way is to use a custom `init`. Generally Codable items in my experience do not have defaults right on the attribute definitions.